__tests__/app.test.js

```diff
@@ -1,17 +1,30 @@
 const app = require('../app')
 const supertest = require('supertest')
 
+// beforeAll(done => {
+//     done()
+//   })
+
+// afterAll(done => {
+//     app.close();
+//     done();
+// });
+
```

Comment:
You'll still need to `seed` before each test and end the db connection afterAll tests here